go.mod

```diff
@@ -1,15 +1,16 @@
 module github.com/GoogleContainerTools/kaniko
 
-go 1.17
+go 1.19
 
 require (
 	cloud.google.com/go/storage v1.29.0
 	github.com/Azure/azure-storage-blob-go v0.14.0
+	github.com/GoogleCloudPlatform/docker-credential-gcr v1.5.1-0.20210909233216-4cdd60d0f2d8
 	github.com/aws/aws-sdk-go v1.44.24
-	github.com/awslabs/amazon-ecr-credential-helper/ecr-login v0.0.0-20220228164355-396b2034c795
+	github.com/awslabs/amazon-ecr-credential-helper/ecr-login v0.0.0-20230322223720-077b4a917a90
```

Comment:
The version of `ecr-login` that was specified on this line differed from what was in the `Dockerfile`.  Rather than trying to guess which one is correct, I went ahead and updated to latest.